Tests/js/views/actions/assets/genericbuttonactionview-tests.js

```diff
@@ -74,6 +74,12 @@ YUI.add('ez-genericbuttonactionview-tests', function (Y) {
             var that = this,
                 actionFired = false;
 
+            this.view.get('container').once('tap', function (e) {
+                Y.Assert.isTrue(
+                    !!e.prevented,
```

Comment:
because e.prevented is an integer (1 in this case), so I do that to be able to use Assert.isTrue